commands/applications.go

```diff
@@ -21,9 +21,9 @@ func NewApplications(network *net.Network) (cmd *Applications) {
 
 func (cmd *Applications) Metadata() command_metadata.CommandMetadata {
 	return command_metadata.CommandMetadata{
-		Name:        "applications",
+		Name:        "application",
 		Description: "Show the status and location of running applications",
-		Usage:       "BROOKLYN_NAME applications",
```

Comment:
Could well be, or even maybe worth renaming it as ApplicationSubCommand or some such.  However, given discussions yesterday we may well not be sticking with "verb noun" style, so this branch may be superseded.